src/App.tsx

```diff
@@ -35,6 +36,10 @@ const App = () => {
     const [todayTodo, setTodayTodo] = useState<Todo[]>([]);
     const [tomorrowTodo, setTomorrowTodo] = useState<Todo[]>([]);
     const [thisWeekTodo, setThisWeekTodo] = useState<Todo[]>([]);
+
+    /* ----------------------------------------------*/
```

Comment:
Never leave unnecessary comments in the code, always try to match with the way the other code style.